src/v2/ion/uiSchemaTransformer.js

```diff
@@ -56,7 +56,10 @@ const createUISchema = (remediationValue = []) => {
       };
     }
     //select factor form for multiple factor enroll and multiple factors verify
-    if (ionFormField.name === 'factorType' || ionFormField.name === 'factorId') {
+    // when factor has not been enrolled we get back factorProfileId form the API
+    if (ionFormField.name === 'factorType' ||
+      ionFormField.name === 'factorId' ||
+      ionFormField.name === 'factorProfileId') {
```

Comment:
are all three still relevant or can choose specific ones?

Reply:
Like my comment describes we'll need `factorId ` and `factorProfileId `. Waiting to confirm with API folks to see if we can remove check on `factorType `.